css/src/components/tabs.scss

```diff
@@ -0,0 +1,115 @@
+$tabs-padding: $font-size-9 $font-size-5 !default;
+$tabs-padding-lg: $font-size-7 $font-size-4 !default;
+
+$tab-font-color: $primary !default;
+$tab-border: $border-width solid transparent !default;
+$tabs-border: $border-width solid $border !default;
+$tabs-hover-background-color: $primary-background !default;
+$tabs-hover-border: $border-width solid $border !default;
+$tabs-hover-border-color: $primary-hover !default;
+$tabs-active-hover-background-color: $primary-hover !default;
+$tabs-active-hover-border-color: $primary-hover !default;
+$tabs-active-background-color: $primary-active;
+$tabs-active-border-color: $primary-active !default;
+$tabs-active-color: $text-invert !default;
+
+.tabs {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $font-size-7;
+	grid-template-columns: max-content auto max-content;
+	white-space: nowrap;
+}
+
+.tab-previous,
+.tab-next {
+	display: flex;
+	align-items: center;
+	padding: $tabs-padding;
+	border: $tab-border;
+	border-radius: $border-radius-rounded;
+	background-color: $tabs-active-background-color;
+	color: $tabs-active-color;
+	font-weight: $weight-semibold;
+	box-shadow: $box-shadow-medium;
+
+	&:hover {
+		border-color: $tabs-active-hover-border-color;
+		background-color: $tabs-active-hover-background-color;
+
+		@include forced-colors {
+			border-color: Highlight;
+			background-color: Highlight;
+			color: HighlightText;
+			forced-color-adjust: preserve-parent-color;
+		}
+	}
+}
+
+.tab-list {
+	grid-column: 2;
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	border-radius: $border-radius-rounded;
+	box-shadow: $box-shadow-medium;
+	overflow-x: hidden;
+}
+
+.tab-control {
+	display: flex;
+	position: relative;
+	align-items: center;
+	justify-content: center;
+	padding: $tabs-padding;
+	border: $tab-border;
+	border-radius: $border-radius-rounded;
+	background-color: transparent;
+	color: $tab-font-color;
+	font-weight: $weight-semibold;
+	cursor: pointer;
+	appearance: none;
+	vertical-align: top;
+
+	&:hover {
+		border-color: $tabs-hover-background-color;
+		background-color: $tabs-hover-background-color;
+		z-index: $zindex-hover;
+
+		@include forced-colors {
+			border-color: Highlight;
+			background-color: Highlight;
+			color: HighlightText;
+			forced-color-adjust: preserve-parent-color;
+		}
+	}
+
+	&[aria-selected='true'] {
```

Comment:
We don't typically use accessibility attrs for styling. Shall we change this to `is-active`?

Reply:
that problem with this is that the `tab-containter` uses the aria-selected to update the styles. if need to add the `.is-active` I would also need to add more functionality for this. At that point its more like not even using tab-container in the first place

Reply:
Personally, [I really like using ARIA for styles](https://benmyers.dev/blog/semantic-selectors/) 😛

Reply:
Okay, that's fine with me, don't need to be a stick in the mud here